src/common.js

```diff
@@ -88,8 +88,7 @@ async function runKeyguard(RequestApiClass, opts) { // eslint-disable-line no-un
 
     // Back arrow functionality
     document.body.addEventListener('click', event => {
-        // @ts-ignore (Property 'matches' does not exist on type 'EventTarget'.)
-        if (!event.target || !event.target.matches('a.page-header-back-button')) return;
+        if (!(event.target instanceof HTMLElement && event.target.matches('a.page-header-back-button'))) return;
```

Comment:
I liked the previous double-negative OR syntax better.

Reply:
Me theoretically too, I only changed it to make it fit into a single line.

Reply:
Changed it back to double negative, that version is only one character longer and still fits into one line.

Reply:
Oh, right :+1:
Sometimes in my IDE lines look longer when it shows names of the passed parameters.